client.go

```diff
@@ -69,6 +67,11 @@ func (c *Client) Logoff() error {
 	return c.s.logoff(c.ctx)
 }
 
+// TargetInfo returns the target info obtained during the NTLMSSP negotation
```

Comment:
How about implementing 
```
func (i *NTLMInitiator) TargetInfo() *NTLMTargetInfo {
}
```
instead of this? So that we don't have to change error handling part, I guess.
Also re-defining `ntlm.SessionTargetInfo` as `NTLMTargetInfo` in this package might be better than exposing internal package.

Reply:
Good idea - I will add it to the Initiator type

I agree on not wanting to expose internal packages, but I struggled to find a way to do it without an import cycle (ntlm can't import smb2.NTLMTargetInfo). I'll see if I can think of a different way